rpc/params/params.go

```diff
@@ -759,6 +759,8 @@ type StateServingInfo struct {
 	// this will be passed as the KeyFile argument to MongoDB
 	SharedSecret   string `json:"shared-secret"`
 	SystemIdentity string `json:"system-identity"`
+	// SSHServerHostKey is the SSH server host key for the embedded SSH server.
+	SSHServerHostKey string `json:"ssh-server-host-key"`
```

Comment:
This should be `omitempty` so that we can ensure that we support 3.6 agents in 3.6.x controllers.